helix-term/src/commands.rs

```diff
@@ -2438,6 +2440,26 @@ fn last_picker(cx: &mut Context) {
     }));
 }
 
+fn goto_next_from_last_picker(cx: &mut Context) {
+    cx.callback = Some(Box::new(
+        move |compositor: &mut Compositor, cx: &mut compositor::Context| {
+            if let Some(picker) = &mut compositor.last_picker {
+                picker.as_pickstepper().unwrap().next(cx);
```

Comment:
I'm sad about adding a method to Component for this, but I couldn't see a better way to do it.

I was able to get it to work without adding a method with this replacing `as_pickstepper`:

```
                if let Some(fp) = picker
                    .as_any_mut()
                    .downcast_mut::<Overlay<FilePicker<PathBuf>>>()
                {
                    fp.content.next(cx);
                }
```

but saying that "works" is stretching it, especially since we'd need to add another block for each possible Picker type.

Maybe someone who knows more Rust has a better idea?